generators/server/templates/src/main/docker/keycloak.yml.ejs

```diff
@@ -21,16 +21,13 @@ version: '<%= DOCKER_COMPOSE_FORMAT_VERSION %>'
 services:
   keycloak:
     image: <%= DOCKER_KEYCLOAK %>
-    command: ["-b", "0.0.0.0", "-Dkeycloak.migration.action=import", "-Dkeycloak.migration.provider=dir", "-Dkeycloak.migration.dir=/opt/jboss/keycloak/realm-config", "-Dkeycloak.migration.strategy=OVERWRITE_EXISTING", "-Djboss.socket.binding.port-offset=1000", "-Dkeycloak.profile.feature.upload_scripts=enabled"]
+    command: ["start-dev", "-Dkeycloak.migration.action=import", "-Dkeycloak.migration.provider=dir", "-Dkeycloak.migration.dir=/opt/jboss/keycloak/realm-config", "-Dkeycloak.migration.strategy=OVERWRITE_EXISTING", "-Dkeycloak.profile.feature.scripts=enabled"]
```

Comment:
I don't see `"start-dev` command in docker-compose file. What's the purpose of this command?

Reply:
the whole image has changed as you might know, they switched to quarkus. Did you check the link the issue comments.

Reply:
sorry, I don't have bandwidth to look into details. Your reply hasn't addressed my question

Reply:
In that case, don't we require it in the docker-compose file also?

Reply:
Oh clearly, that was a mistake, it should be present in app.yml two, just fixed it :)